vcd/resource_vcd_inserted_media_test.go

```diff
@@ -196,25 +197,27 @@ resource "vcd_vapp_vm" "{{.VmName}}" {
   depends_on    = ["vcd_vapp.{{.VappName}}"]
 }
 
-resource "vcd_catalog_media"  "{{.CatalogMediaName}}" {
-  org = "{{.Org}}"
+resource "vcd_catalog_media" "{{.CatalogMediaName}}" {
+  org     = "{{.Org}}"
   catalog = "{{.Catalog}}"
 
-  name = "{{.CatalogMediaName}}"
-  description = "{{.Description}}"
-  media_path = "{{.MediaPath}}"
-  upload_piece_size = {{.UploadPieceSize}}
+  name                 = "{{.CatalogMediaName}}"
+  description          = "{{.Description}}"
+  media_path           = "{{.MediaPath}}"
+  upload_piece_size    = {{.UploadPieceSize}}
   show_upload_progress = "{{.UploadProgress}}"
 }
 
 resource "vcd_inserted_media" "{{.InsertMediaName}}" {
-  org          = "{{.Org}}"
-  vdc          = "{{.Vdc}}"
+  org     = "{{.Org}}"
+  vdc     = "{{.Vdc}}"
   catalog = "{{.Catalog}}"
-  name = "{{.CatalogMediaName}}"
-  
-  vapp_name = "${vcd_vapp.{{.VappName}}.name}" 
-  vm_name = "${vcd_vapp_vm.{{.VmName}}.name}" 
+  name    = "{{.CatalogMediaName}}"
+
+  vapp_name  = "${vcd_vapp.{{.VappName}}.name}"
+  vm_name    = "${vcd_vapp_vm.{{.VmName}}.name}"
   depends_on = ["vcd_vapp_vm.{{.VmName}}", "vcd_catalog_media.{{.CatalogMediaName}}"]
+
+  eject_force = "{{.ejectForce}}"
```

Comment:
Typo
```suggestion
  eject_force = "{{.EjectForce}}"
```

Currently the test `.tf` file reenders so as it misses the value:
```
$  cat vcd/test-artifacts/vcd.TestAccVcdMediaInsertBasic.tf | tail -5
  vm_name    = "${vcd_vapp_vm.TestAccVcdVAppVmForInsert.name}"
  depends_on = ["vcd_vapp_vm.TestAccVcdVAppVmForInsert", "vcd_catalog_media.TestAccVcdCatalogMediaBasicForInsert"]

  eject_force = ""
}
```

Reply:
Well in sense we don't change default behaviour. Currently with tuning VM you usually stumble on issue.